### ISSUE TITLE
Unable to connect to local SQL Server on Windows 11

### ISSUE THREAD

**Issue by pandenista1970** [dbeaver/dbeaver#36649]
### Description

Trying to connect to SQL Server through Microsoft SQL Server driver ends with error message:
The TCP/IP connection to the host XXX, port 1433 has failed. Error: "Connection refused: no further information. Verify the connection properties. Make sure that an instance of SQL Server is running on the host and accepting TCP/IP connections at the port. Make sure that TCP connections to the port are not blocked by a firewall.".
I made a rule in Firewall to allow DBeaver in all types of nets.
I tried to connect from other application like Azure Data Studio or SQL Server Management Studio and everything connects there fine.

### DBeaver Version

Community edition 24.3.1

### Operating System

Windows 11 Pro 24H2

### Database and driver

SQL Server 16.0.1135.2

### Steps to reproduce

1. I choose the driver SQL Server
2. Set the connection details as on the image below
3. Download required driver (if for the first time)
4. Try connection
5. Error message described in the description appears in a moment

![obrazek](https://github.com/user-attachments/assets/9c9d4fac-4ddf-4a3a-8893-9cbf51541541)

**Comment by flambchap** [comment]
Hi, @pandenista1970. I see that you specified the host as `localhost`. Is your database running on your local machine?

**Comment by pandenista1970** [comment]
Yes, it's a local database. It doesn't work either if I use my computer name as a server.

**Comment by ask9** [comment]
if your sql server is installed as a named instance, you must define it in host, for example:
localhost\your_named_instance

**Comment by pandenista1970** [comment]
If I use this, it fails on another error message (if the name of my instance is XXX):
The connection to the host localhost, named instance XXX failed. Error: "java.net.SocketTimeoutException: Receive timed out". Verify the server and instance names and check that no firewall is blocking UDP traffic to port 1434. For SQL Server 2005 or later, verify that the SQL Server Browser Service is running on the host.

**Comment by ask9** [comment]
Check if you can connect via SSMS and what connection properties you use there (windows auth or etc).
Check in SQL Server Configuration Manager if your SQL Server instance is running.
Check if your instance makes use of default port or another port.
It doesn't seem the issue is in DBeaver.

**Comment by pandenista1970** [comment]
As I wrote before in the description everything works well in SSMS. I used the same configuration. The problem must be somewhere in the driver.

**Comment by ShadelessFox** [comment]
Hello @pandenista1970,

Is the issue still relevant to you?

Can you please provide log files? You can find instructions here: https://github.com/dbeaver/dbeaver/wiki/Log-files

**Comment by E1izabeth** [comment]
It's been a while since there has been an update here. Please let me know if the issue is still present and provide additional information.

**Comment by pandenista1970** [comment]
Thougth I install regularly new versions, there is no progress in this issue - it fails with the same error messages. I sent the error logs through e-mail but I add it here, too.

[DBeaverErrorLog.log](https://github.com/user-attachments/files/19195825/DBeaverErrorLog.log)

**Comment by MashaKorax** [comment]
Hello @pandenista1970 . I reproduce your issue. But the issue is not on DBeaver's side. You should open SQL Server Management Studio (SSMS), go to SQL Server Network Configuration, and enable TCP/IP (**it is disabled by default!**). After that restart the SQL server 

Finally, go to Dbeaver and choose Windows authentication.

Was it helpful for you? Please let me know
 

![Image](https://github.com/user-attachments/assets/1ddcc907-e738-4905-8073-2b00c459045f)


![Image](https://github.com/user-attachments/assets/345aa7d1-9230-4b3a-b41c-bfde94a2c9ee)

![Image](https://github.com/user-attachments/assets/0e5fb382-a022-40db-a8c1-18767406beae)
